server/helpers.js

```diff
@@ -119,3 +121,13 @@ export function isCredentialError(err) {
            err.errCode === getErrorCode('INVALID_PARAMETERS') ||
            err.errCode === getErrorCode('NO_PASSWORD');
 }
+
+export function setupMoment(locale) {
+    if (locale) {
```

Comment:
could do that (final proposal. But I wanted to keep the possibility to manipulate the moments in an understandable way